### DISCUSSION TITLE
Could not get unknown property 'linkTask' for object of type org.gradle.language.cpp.internal.DefaultCppStaticLibrary

### DISCUSSION THREAD

**Discussion by pradeepsimba** [nokeedev/gradle-native#608] [Q&A]
**Could not get unknown property 'linkTask' for object of type org.gradle.language.cpp.internal.DefaultCppStaticLibrary**

link : [error][1]

[error][1] at this [line][2].

Why this error comes?

How can I solve this error ?

  [1]: https://github.com/Jeevasimba/windowsnative/runs/6258866081?check_suite_focus=true#step:7:76
  [2]: https://github.com/Jeevasimba/windowsnative/blob/main/hello/build.gradle#L29

**Comment by lacasseio** [comment]
In this case, you are after the `createTask`. Shared libraries, executables and bundles are *linked* while static libraries are *created*. Static libraries are *linked* together (among other objects, etc.) to create shared libraries, executables and/or bundles.

The same applies to the models in the Nokee plugins.

**Comment by lacasseio** [comment]
I converted the issue into a discussion that better suits the request.
